sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisAsyncClientTest.java

```diff
@@ -1384,9 +1393,9 @@ public void analyzeContentUrlWithQueryFields(HttpClient httpClient, DocumentAnal
         }, INVOICE_PDF);
     }
 
+    @RecordWithoutRequestBody
```

Comment:
This is my first time seeing it. Why want to request without body?

Reply:
Because we don't really care for the request body and it is huge. 
It is a new feature to record without request body using Test proxy/